.markdown-lint.yml

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I suggest moving this file to `./github/linters`.  See my comment on `ci.yml` for more details.